server.js

```diff
@@ -38,28 +36,117 @@ var renderHtml = function(url, cb) {
             }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
         });
     };
+
+    console.debug("Opening page");
     page.open(url);
 
     page.onError = function (msg, trace) {
-        console.log(msg);
+        console.error(msg);
         trace.forEach(function(item) {
-            console.log('  ', item.file, ':', item.line);
+            console.error('  ', item.file, ':', item.line);
         });
     };
 };
 
+var configure = function() {
+    console.info("Reading config.json");
+
+    var content = fs.read('config.json');
+    var settings = JSON.parse(content);
+
+    var options = {}
+
+    if (fs.exists('/etc/prerender.json')) {
+        console.info("Reading /etc/prerender.json");
+        var content = fs.read('/etc/prerender.json');
+        var options = JSON.parse(content);
+    }
+    settings = mergeOptions(settings, options);
+
+    if (fs.exists('config-local.json')) {
+        console.info("Reading config-local.json");
+        var content = fs.read('config-local.json');
+        var options = JSON.parse(content);
+    }
+    settings = mergeOptions(settings, options);
+
+    return settings;
+}
+
+var mergeOptions = function(obj1, obj2) {
```

Comment:
You can use "extend" npm module for that. It's 95% same code but already written for you